ClearKeep/Scenes/ChatGroup/Views/CreateGroupView.swift

```diff
@@ -0,0 +1,182 @@
+					.font(AppTheme.shared.fontSet.font(style: .body1))
+					.foregroundColor(foregroundButtonBack)
+			}
+			.frame(maxWidth: .infinity, alignment: .leading)
+		}
+	}
+
+	var groupNameView: some View {
+		VStack(alignment: .leading) {
+			Text("GroupChat.Group.Name".localized)
+				.font(AppTheme.shared.fontSet.font(style: .body2))
+				.foregroundColor(foregroundButtonBack)
+
+			CommonTextField(text: $nameGroup,
+							inputStyle: $nameGroupStyle,
+							inputIcon: Image(""),
+							placeHolder: "GroupChat.Named".localized,
+							keyboardType: .default,
+							onEditingChanged: { isEditing in
+				if isEditing {
+					nameGroupStyle = .normal
+				} else {
+					nameGroupStyle = .highlighted
+				}
+			})
+			Text("GroupChat.User.Inside".localized)
+				.foregroundColor(foregroundText)
+				.font(AppTheme.shared.fontSet.font(style: .input2))
+		}
+	}
+
+	var listUserView: some View {
+		List(model, id: \.id) { item in
+			HStack {
+				ZStack {
+					Circle()
+						.fill(backgroundGradientPrimary)
+						.frame(width: Constants.sizeImage, height: Constants.sizeImage)
+					AppTheme.shared.imageSet.userIcon
+				}
+				Text(item.name)
+					.frame(minWidth: 0, maxWidth: .infinity, alignment: .leading)
+					.font(AppTheme.shared.fontSet.font(style: .input2))
+					.foregroundColor(Color.gray)
+			}
+		}
+		
+			.listStyle(PlainListStyle())
+		
+	}
+
+	var buttonNextView: some View {
+		NavigationLink(
+			destination: EmptyView(),
+			label: {
+				Button(action: { },
+					   label: {
+					Text("GroupChat.Create".localized)
+				})
+					.frame(maxWidth: .infinity)
+					.frame(height: Constants.heightButton)
+					.font(AppTheme.shared.fontSet.font(style: .body3))
+					.background(backgroundGradientPrimary)
+					.foregroundColor(AppTheme.shared.colorSet.offWhite)
+					.cornerRadius(Constants.cornerRadiusButtonNext)
+					.padding(.horizontal, Constants.spacerTopView)
+			})
+			.padding(.bottom, Constants.paddingButtonNext)
+	}
+}
+
+// MARK: - Color func
+private extension CreateGroupView {
+	var foregroundButtonBack: Color {
+		colorScheme == .light ? AppTheme.shared.colorSet.black : AppTheme.shared.colorSet.offWhite
+	}
+
+	var backgroundGradientPrimary: LinearGradient {
+		LinearGradient(gradient: Gradient(colors: AppTheme.shared.colorSet.gradientPrimary), startPoint: .leading, endPoint: .trailing)
+	}
+
+	var backgroundButtonBack: [Color] {
+		colorScheme == .light ? [AppTheme.shared.colorSet.offWhite, AppTheme.shared.colorSet.offWhite] : [AppTheme.shared.colorSet.black, AppTheme.shared.colorSet.black]
+	}
+
+	var foregroundText: Color {
+		colorScheme == .light ? AppTheme.shared.colorSet.grey3 : AppTheme.shared.colorSet.greyLight
+	}
+}
+
+// MARK: - Private Func
+private extension CreateGroupView {
+	func customBack() {
+		self.presentationMode.wrappedValue.dismiss()
+	}
+}
+
+// MARK: - Preview
+#if DEBUG
+struct CreateGroupView_Previews: PreviewProvider {
+	static var previews: some View {
+		CreateGroupView()
+    }
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/db64d1bb87033938f80a14c709ea4da6f861ca2c/ClearKeep/Scenes/ChatGroup/Views/CreateGroupView.swift#L180"></span>Please use tabs instead of spaces<br />
<code>not_spaces</code> <code>CreateGroupView.swift:180</code></td>
    </tr>
  </tbody>
</table>